schemas/common/page.schema.json

```diff
@@ -0,0 +1,67 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/page",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Page Object",
+  "type": "object",
+  "description": "Page object contains paging information for related resource when resource result is paginated.",
+  "definitions": {
+    "page": {
+      "properties": {
+        "orderBy": {
+          "title": "Order by property",
+          "type": "string",
+          "description": "The property by which this page is ordered.\n"
+        },
+        "order": {
+          "title": "Sort order",
+          "type": "string",
+          "pattern": "^(asc|desc)$",
```

Comment:
Better: `"enum": [ "asc", "desc" ]` and 
`"meta:enum": {"asc": "Ascending order, i.e. the entries with the greatest value according to `orderBy` come first.", "desc": "Descending order"}`

Reply:
+1 for `meta:enum`, though this property is likely going away.